app/js/checkAnswers.js

```diff
@@ -129,3 +129,83 @@ function displayResult(earnedPoints, earnedPercentage, answeredQuestions) {
     document.querySelector(".answered_questions").innerHTML = answeredQuestions
     document.querySelector(".score_percentage").innerHTML = earnedPercentage
 }
+
+/**
+ * function adds event listeners to .question and listens for click event within here
+ * it then updates the class of the span containing the question number allowing styling to be applied
+ *
+ */
+function addAnswerEventListeners() {
+    document.querySelectorAll('.question').forEach(function (input) {
+        input.addEventListener('click', function(e) {
+            if (e.target.tagName == 'INPUT') {
+                let nav = document.querySelector('#question-nav')
```

Comment:
nav variable isnt needed as your only using it once

Reply:
changed in 8f1ccfa